packages/sit-onyx/src/styles/mixins/normalize.scss

```diff
@@ -0,0 +1,221 @@
+@mixin apply {
+  html:focus-within & {
+    scroll-behavior: smooth;
+  }
+
+  &,
+  * {
+    &,
+    &::before,
+    &::after {
+      margin: 0;
+
+      box-sizing: border-box;
+
+      -moz-text-size-adjust: none;
+      -webkit-text-size-adjust: none;
+      text-size-adjust: none;
+    }
+
+    &:is(ul, ol) {
+      list-style: none;
+    }
+
+    &:is(input, button, textarea, optgroup, select) {
+      font: inherit;
+    }
+
+    /*! Based on modern-normalize v2.0.0 | MIT License | https://github.com/sindresorhus/modern-normalize */
+
+    /*
+    Grouping content
+    ================
+    */
+
+    /**
+    1. Add the correct height in Firefox.
+    2. Correct the inheritance of border color in Firefox. (https://bugzilla.mozilla.org/show_bug.cgi?id=190655)
+    */
+
+    &:is(hr) {
+      height: 0; /* 1 */
```

Comment:
This will hide the element, right? Why would we want this?

Reply:
updated comment